ee/clickhouse/queries/breakdown_props.py

```diff
@@ -142,12 +142,14 @@ def format_breakdown_cohort_join_query(team_id: int, filter: Filter, **kwargs) -
     return " UNION ALL ".join(cohort_queries), ids, params
 
 
-def _parse_breakdown_cohorts(cohorts: BaseManager) -> Tuple[List[str], Dict]:
+def _parse_breakdown_cohorts(cohorts: Union[BaseManager, List[Cohort]]) -> Tuple[List[str], Dict]:
```

Comment:
`BaseManager` is most certainly the wrong type here. 

I guess you want to pass in a `QuerySet` here - you could `Union[QuerySet[Cohort], List[Cohort]]` but honestly wrapping all callsites in `list()` seems more sensible.